src/app/base/components/node/StorageTables/AvailableStorageTable/BulkActions/CreateRaid/CreateRaid.test.tsx

```diff
@@ -122,17 +121,15 @@ describe("CreateRaid", () => {
       },
       payload: {
         params: {
-          block_devices: [1, 2],
+          block_devices: [9],
           fstype: "ext4",
-          level: "raid-6",
+          level: "raid-1",
           mount_options: "option1,option2",
           mount_point: "/mount-point",
           name: "md1",
-          partitions: [3, 4],
-          spare_devices: [5, 6],
-          spare_partitions: [7, 8],
+          partitions: [10],
```

Comment:
Related tot he change suggested above.

The ids in `partitions` and `block_devices` come from the initial state of this component. Would be better to explicitly specify it to make it clear where this comes from (and more resilient to change).

Otherwise the id will change making the test fail (somewhat unexpectedly) e.g. if you add a new test case as the `id` is incremented sequentially.